deploy/kubernetes/dolphinscheduler/values.yaml

```diff
@@ -141,6 +141,9 @@ security:
         truststorepassword: ""
 
 conf:
+  # auto restart, if true, all components will be restarted automatically after the common configuration is updated. if false, you need to restart the components manually. default is false
+  auto: false
```

Comment:
Basically I share the same opinions with @Radeity @SbloodyS @qingwli . But I think it is fine if we have a configuration here and set to false by default.